keyboards/annepro2/info.json

```diff
@@ -3,7 +3,7 @@
     "url": "https://openannepro.github.io/",
     "maintainer": "bwisn",
     "usb": {
-        "vid": "0xFEED",
+        "vid": "0x04D9",
```

Comment:
Technically, this looks to be holtek's VID. `0xAC20` would be better, I think.

Reply:
Yeah, looks like it, however it looks like Obins borrowed the VID, I even found it referenced in SDL - https://github.com/libsdl-org/SDL/blob/main/src/joystick/SDL_joystick.c

So if we want to keep the same VID/PID as the original keyboard, I think 0x04D9 is fine

However I'd understand if you want to change the VID to differentiate between the original keyboard and one with QMK

Reply:
@drashna I've changed the VID assuming we want to differentiate QMK from the original keyboard (if we don't want to differentiate, we should switch it back)